src/main/java/com/amazonaws/services/kinesis/clientlibrary/lib/worker/ShardConsumer.java

```diff
@@ -420,6 +430,7 @@ private ITask getNextTask() {
     void updateState(TaskOutcome taskOutcome) {
         if (taskOutcome == TaskOutcome.END_OF_SHARD) {
             markForShutdown(ShutdownReason.TERMINATE);
+            LOG.info("Shard " + shardInfo.getShardId() + "Mark for shutdown with reason TERMINATE");
```

Comment:
space or delimiter between shardId and Mark

Reply:
Addressed